packages/components/src/popover/stories/index.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * External dependencies
+ */
+import { boolean, select, text } from '@storybook/addon-knobs';
+
+/**
+ * Internal dependencies
+ */
+import Popover from '../';
+
+/**
+ * WordPress dependencies
+ */
+import { useState, useEffect } from '@wordpress/element';
+
+export default { title: 'Popover', component: Popover };
+
+export const _default = () => {
+	const show = boolean( 'Example: Show', true );
+	const children = text( 'children', 'Popover Content' );
+	const animate = boolean( 'animate', false );
+	const focusOnMount = select(
+		'focusOnMount',
+		{
+			firstElement: 'firstElement',
+			container: 'container',
+		},
+		'firstElement'
+	);
+	const noArrow = boolean( 'noArrow', false );
+
+	const props = {
+		animate,
+		children,
+		focusOnMount,
+		noArrow,
+	};
+
+	if ( ! show ) {
+		return null;
+	}
+
+	return <Popover { ...props } />;
+};
+
+const DragExample = ( props ) => {
```

Comment:
Would it be possible to expose the Drag&Drop mechanism as something generic other stories can use?

Reply:
@jorgefilipecosta Thank you so much!!

> Would it be possible to expose the Drag&Drop mechanism as something generic other stories can use?

I can certainly create something and `export` it. In the past, I used this library for the stories:
https://github.com/mzabriskie/react-draggable

For this PR, I wrote out the bare bones functionality manually to keep it simple :).